tokens/ust.svg

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="1000px" height="1000px" style="shape-rendering:geometricPrecision; text-rendering:geometricPrecision; image-rendering:optimizeQuality; fill-rule:evenodd; clip-rule:evenodd" xmlns:xlink="http://www.w3.org/1999/xlink">
+<g><path style="opacity:0.998" fill="#0e3ca5" d="M 692.5,111.5 C 674.251,102.589 654.918,97.4222 634.5,96C 597.274,94.7461 560.607,98.7461 524.5,108C 449.815,124.854 380.149,153.521 315.5,194C 284.928,213.738 259.928,238.904 240.5,269.5C 216.806,254.473 197.473,235.139 182.5,211.5C 199.609,202.232 216.609,192.732 233.5,183C 254.693,170.154 273.86,154.654 291,136.5C 301.384,125.403 308.384,112.403 312,97.5C 315.499,83.3532 311.333,72.1865 299.5,64C 292.826,60.7193 285.826,58.5526 278.5,57.5C 347.062,26.9191 418.729,16.4191 493.5,26C 568.163,36.02 634.496,64.52 692.5,111.5 Z"/></g>
+<g><path style="opacity:0.995" fill="#5392f6" d="M 278.5,57.5 C 285.826,58.5526 292.826,60.7193 299.5,64C 311.333,72.1865 315.499,83.3532 312,97.5C 308.384,112.403 301.384,125.403 291,136.5C 273.86,154.654 254.693,170.154 233.5,183C 216.609,192.732 199.609,202.232 182.5,211.5C 157.359,222.271 131.692,231.771 105.5,240C 87.8827,245.49 69.8827,248.99 51.5,250.5C 48.2834,250.653 45.2834,249.986 42.5,248.5C 80.0304,173.471 134.697,114.304 206.5,71C 229.142,58.9711 253.142,54.4711 278.5,57.5 Z"/></g>
+<g><path style="opacity:0.999" fill="#5492f6" d="M 692.5,111.5 C 756.485,164.052 800.985,230.052 826,309.5C 829.33,321.157 832.497,332.824 835.5,344.5C 830.553,345.285 825.553,345.951 820.5,346.5C 726.526,361.961 632.526,377.295 538.5,392.5C 538.5,404.833 538.5,417.167 538.5,429.5C 538.5,442.167 538.5,454.833 538.5,467.5C 538.5,479.833 538.5,492.167 538.5,504.5C 538.5,516.833 538.5,529.167 538.5,541.5C 538.5,553.833 538.5,566.167 538.5,578.5C 538.5,591.167 538.5,603.833 538.5,616.5C 537.501,542.002 537.167,467.335 537.5,392.5C 506.836,397.074 476.17,401.407 445.5,405.5C 434.816,406.354 424.149,407.354 413.5,408.5C 393.365,410.52 373.365,412.52 353.5,414.5C 343.791,414.741 334.124,415.408 324.5,416.5C 307.845,417.44 291.178,418.107 274.5,418.5C 256.124,400.942 242.291,380.276 233,356.5C 223.144,326.421 225.644,297.421 240.5,269.5C 259.928,238.904 284.928,213.738 315.5,194C 380.149,153.521 449.815,124.854 524.5,108C 560.607,98.7461 597.274,94.7461 634.5,96C 654.918,97.4222 674.251,102.589 692.5,111.5 Z"/></g>
```

Comment:
The icon for the UST token is too huge, it seems it was converted from PNG, but it's too big to be set as metadata. Please find another SVG or make a conversion using another tool and simplifications (image size, number of colors, etc)

Reply:
updated, please let me know if this size works!

Reply:
Thanks, still quite big, but works for now. But if you find a small (non-converted) SVG, please create another one PR.

Reply:
will do soon :+1: